activesupport/lib/active_support/core_ext/range/include_range.rb

```diff
@@ -1,25 +1,7 @@
 # frozen_string_literal: true
 
-module ActiveSupport
-  module IncludeWithRange #:nodoc:
-    # Extends the default Range#include? to support range comparisons.
-    #  (1..5).include?(1..5) # => true
-    #  (1..5).include?(2..3) # => true
-    #  (1..5).include?(2..6) # => false
-    #
-    # The native Range#include? behavior is untouched.
-    #  ('a'..'f').include?('c') # => true
-    #  (5..9).include?(11) # => false
-    def include?(value)
-      if value.is_a?(::Range)
-        # 1...10 includes 1..9 but it does not include 1..10.
-        operator = exclude_end? && !value.exclude_end? ? :< : :<=
-        super(value.first) && value.last.send(operator, last)
-      else
-        super
-      end
-    end
-  end
-end
+ActiveSupport::Deprecation.warn "You have required `active_support/core_ext/range/include_range`. " \
```

Comment:
[Every](https://github.com/rails/rails/blob/6c574ac58993512f975ddaf1f679c5956cc576df/activesupport/lib/active_support/core_ext/hash/compact.rb#L3) [other](https://github.com/rails/rails/blob/6c574ac58993512f975ddaf1f679c5956cc576df/activesupport/lib/active_support/core_ext/hash/transform_values.rb#L3) [file](https://github.com/rails/rails/blob/6c574ac58993512f975ddaf1f679c5956cc576df/activesupport/lib/active_support/core_ext/numeric/inquiry.rb#L3) in Active Support that emits a deprecation requires `active_support/deprecation` first; we should do the same here.

Reply:
Corrected.